### PR TITLE
fixed style of filter box

### LINKED ISSUE #451: Fix Filtering Style
image

### PR DESCRIPTION
**Pull request by isabellecelo**
Screenshot 2023-09-04 at 3 12 08 PM
fixed #451

### CODE REVIEW

File: src/components/Admin/DropDown.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const Menu = ({ setOption, className, setOptions, options }) => {
     <div className={className}>
       <input
         autoFocus
-        className="mx-3 my-2 w-11/12 ring-0 outline-none border-gray-300 border-[1px] text-sm px-2 py-1"
+        className="mx-1.5 my-1 w-11/12 ring-0 outline-none border-transparent !rounded-b-xl !rounded-t-xl border-[1px] px-2 py-1"
```
Comment (shahdivyank) [inline src/components/Admin/DropDown.jsx]: dont need the !

Comment (shahdivyank) [reply src/components/Admin/DropDown.jsx]: should be able to do rounded-y-xl

**Comment by shahdivyank** [comment]
Add labels

File: src/components/Admin/DropDown.jsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Menu = ({ setOption, className, setOptions, options }) => {
     <div className={className}>
       <input
         autoFocus
-        className="mx-3 my-2 w-11/12 ring-0 outline-none border-gray-300 border-[1px] text-sm px-2 py-1"
+        className="mx-1.5 my-1 w-11/12 ring-0 outline-none border-transparent rounded-y-xl border-[1px] px-2 py-1"
```
Comment (shahdivyank) [inline src/components/Admin/DropDown.jsx]: u can remove the border entirely if its gonna be transparent

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**